.vscode/launch.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
These two vscode files are user specific. Can we please remove them from the check-in?

Reply:
I updated the .gitignore and should have the .vscode directory removed. Let me know if that meets the goal, if not, I'll ask you to decline the pull request so that I may rebase the code and submit another pull request with a "cleaner" path.

Reply:
@aluevano it looks like the files still exist. Perhaps, it is best to create a new PR?